src/tasks/AutoCinemaTask.py

```
@@ -0,0 +1,163 @@
+    def select_date(self):
+        have_date = False
+        regex_str = self.config.get(self.CONF_MOVIE_CHAR,"薄荷")
+        match_regex = re.compile(regex_str)
+        while self.enabled and not have_date:
+            if self.ocr(0.77, 0.22, 0.9, 0.86, match=match_regex):
+                have_date =True
+                self.sleep(1.14)
+                btn_date = self.wait_ocr(0.77, 0.22, 0.9, 0.86, match=match_regex, time_out=1.14)
+                confirm_range = (0.6465, 0.6125, 0.7047, 0.7049)
+                self.wait_until(
+                    lambda: not self.find_confirm(confirm_range),
```

Ruff (0.15.17)

[warning] Function definition does not bind loop variable `confirm_range`

(B023)